src/rules/ResourceQuoteStyle.js

```diff
@@ -52,6 +52,8 @@ class ResourceQuoteStyle extends ResourceRule {
             this.localeOnly = true;
         }
         this.link = "https://github.com/ilib-js/i18nlint/blob/main/docs/resource-quote-style.md";
+        // According to the MS Style guidelines, quotes are usually not required in Swedish when the source English text contains quotes
+        this.skipLocales = new Set(["sv"]);
```

Comment:
Original requirement here was that the quote style check should not produce an error when no quotes are in target, but still check style when they are

Reply:
Okay, removed the fix and set it back the way it was before.